utility/gccversion.hpp

```diff
@@ -38,7 +38,7 @@
 #endif
 
 // pre-4.7 gcc has no override/final modifier
-#if GCC_VERSION && (GCC_VERSION < 40700)
+#if GCC_VERSION && (GCC_VERSION < 40700) && !defined(__clang__)
```

Comment:
I believe this and the previous macro can go away right? This is pre gcc 4.7... @vaclavblazek

Reply:
Yes, please remove completely.